flow/benchmarks/rllib/es_runner.py

```diff
@@ -60,6 +110,6 @@
             "max_failures": 999,
             "stop": {"training_iteration": 500},
             "num_samples": 1,
-            "upload_dir": "s3://<BUCKET NAME>"
+            "upload_dir": "s3://" + upload_dir
```

Comment:
what if you're not running experiments on aws? this should be defaulted to no uploading there